src/theme/theme.css

```diff
@@ -46,6 +46,24 @@ body {
     /* padding: .4rem; */
 }
 
+/* Should be used for right-click/context/pop-up menus */
+.MENU-ROOT {
```

Comment:
(side note for fun: I usually capital-case stuff when I know my naming is shit, or when I'm really unhappy with what I did, so my code can scream at me until I rework it later 😄
feel free to do whatever feels best for you, you can also split those stuff outside of themes, or add them inline the respective `...UI.tsx` files; whatever works best for you
)